src/tox_travis/after.py

```diff
@@ -81,10 +80,10 @@ def travis_after():
     print('All required jobs were successful.')
 
 
-def after_config_matches():
+def after_config_matches(config):
     """Determine if this job should wait for the others."""
-    config = py.iniconfig.IniConfig('tox.ini')
-    section = config.sections.get('travis:after', {})
+    tox_ini = get_iniconfig(config)
+    section = tox_ini.sections.get('travis:after', {})
```

Comment:
Good point. Wasn't aware that `setup.cfg` was even a thing. 

I'm not partial to any particular fucntion/variable name - lemme know what you want to do here.